velox/exec/RowContainer.cpp

```diff
@@ -512,4 +504,271 @@ void RowContainer::setProbedFlag(char** rows, int32_t numRows) {
   }
 }
 
+RowTypePtr RowContainer::rowType() const {
+  VELOX_CHECK(aggregates_.empty(), "Aggregate type info not yet implemented");
```

Comment:
@oerling I'm confused by the error message: "Aggregate type info not yet implemented". What does this check mean?